pandas/core/frame.py

```diff
@@ -1997,6 +1998,13 @@ def to_parquet(self, fname, engine='auto', compression='snappy',
             If ``False``, they will not be written to the file. If ``None``,
             the behavior depends on the chosen engine.
 
+            .. versionadded:: 0.24.0
+        partition_cols : list, optional, default None
+            Column names by which to partition the dataset
+            Columns are partitioned in the order they are given
+            The behaviour applies only to pyarrow >= 0.7.0 and fastparquet
+            For other versions, this argument will be ignored.
```

Comment:
Is it actually ignored for older pyarrows? I would have hoped it would raise when pyarrow gets the unrecognized argument.

Reply:
Actually it seems like we raise. Could you update this?

Reply:
done